src/Cns.ts

```diff
@@ -130,6 +132,29 @@ export default class Cns extends EthereumNamingService {
     return await this.getAllRecords(resolverContract, tokenId);
   }
 
+  async twitter(domain: string): Promise<string> {
+    const verificationAddress = '0x9457f96845DA5bF00408AC1837F3eF3D35ce4563';
+    const twitterHandle = await this.record(domain, 'social.twitter.username');
+    const owner = await this.owner(domain);
+    const validationSignature = await this.record(
+      domain,
+      'validation.social.twitter.username',
+    );
+    const message = [domain, owner, 'validation.twitter.name', twitterHandle]
+      .map((value: string) => sha256(value))
+      .reduce((message, hashedValue) => message + hashedValue, '');
+    const signerAddress = recover(message, validationSignature);
+    if (signerAddress !== verificationAddress) {
+      throw new ResolutionError(
+        ResolutionErrorCode.InvalidTwitterVerification,
```

Comment:
@sudoryan Maybe it's better to return twitter and verification status as boolean? Some users might set this value without verification and it's good to return the twitter username in this case.